hub-mirror/hub.py

```diff
@@ -86,29 +86,17 @@ def dynamic_list(self):
         return self._get_all_repo_names(url)
 
     @functools.lru_cache
-    def _get_all_repo_names(self, url):
-        page, per_page = 1, 60
-        api = url + "?page=0&per_page=" + str(per_page)
+    def _get_all_repo_names(self, url, page=1):
+        per_page = 60
+        api = url + f"?page={page}&per_page=" + str(per_page)
```

Comment:
A quick question: the page start is changed from `0` to `1`, start `0` and start`1`, have same result, right?